src_nbgl/ui_signing.h

```diff
@@ -0,0 +1,21 @@
+#ifndef UI_SIGNING_H_
+#define UI_SIGNING_H_
+
+#define SIGN_BUTTON           "Hold to sign"
+#define REJECT_BUTTON         "Reject"
+#define SIGN(msg)             "Sign " msg "?"
+#define REVIEW(msg)           "Review " msg
+#define REJECT(msg)           "Reject " msg
+#define REJECT_QUESTION(msg)  REJECT(msg) "?"
+#define REJECT_CONFIRM_BUTTON "Yes, reject"
+#define RESUME(msg)           "Go back to " msg
+
+typedef enum {
+    UI_SIGNING_POSITION_START = 0,
+    UI_SIGNING_POSITION_REVIEW,
+    UI_SIGNING_POSITION_SIGN
+} e_ui_signing_position;
+
+extern e_ui_signing_position g_position;
```

Comment:
Can I have an explanation + comments on what this does, I don't understand

Reply:
This is to handle the logic of the signing flow, for example when the user changes his mind and comes back from the reject  screen, we need to know on what step he was on.
It could be split into 3 generic steps:
* the first page "Review ..."
* the review page with the actual content
* the signature page "Sign ...?"

Reply:
This really should not be needed 🤔